tests/test_typos_rollout.py

```
@@ -159,6 +180,233 @@ def test_offline_generation_requires_then_reuses_valid_cache(
+    monkeypatch: pytest.MonkeyPatch,
+    headers: dict[str, str],
+    content: bytes,
+    message: str,
+) -> None:
+    """HTTP sources are bounded with and without Content-Length."""
+    repository, _ = prepare_repository(tmp_path)
+    response = FakeHttpResponse(content, headers)
+    monkeypatch.setattr(
+        generator.rollout,
+        "_open_https",
+        lambda *args, **kwargs: response,
```

Ruff (0.16.0)

[warning] Unused lambda argument: `args`

(ARG005)